entity-service/src/test/kotlin/com/egm/stellio/entity/model/EntityTest.kt

```diff
@@ -0,0 +1,80 @@
+package com.egm.stellio.entity.model
+
+import com.egm.stellio.shared.util.JsonLdUtils
+import com.egm.stellio.shared.util.JsonLdUtils.JSONLD_ID
+import com.egm.stellio.shared.util.JsonLdUtils.JSONLD_TYPE
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_CREATED_AT_PROPERTY
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_LOCATION_PROPERTY
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_MODIFIED_AT_PROPERTY
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_OBSERVATION_SPACE_PROPERTY
+import com.egm.stellio.shared.util.JsonLdUtils.NGSILD_OPERATION_SPACE_PROPERTY
+import com.egm.stellio.shared.util.toUri
+import org.junit.jupiter.api.Assertions
+import org.junit.jupiter.api.Test
+import java.time.Instant
+import java.time.ZoneOffset
+
+class EntityTest {
+
+    private val entity = Entity(
+        id = "urn:ngsi-ld:beehive:01".toUri(),
+        type = listOf("Beehive"),
+        createdAt = Instant.now().atZone(ZoneOffset.UTC),
+        modifiedAt = Instant.now().atZone(ZoneOffset.UTC)
+    )
+
+    @Test
+    fun `it should serialize entity without createdAt and modifiedAt if not specified`() {
+        val serializedEntity = entity.serializeCoreProperties(false)
+        Assertions.assertFalse(serializedEntity.contains(NGSILD_CREATED_AT_PROPERTY))
+        Assertions.assertFalse(serializedEntity.contains(NGSILD_MODIFIED_AT_PROPERTY))
+        Assertions.assertFalse(serializedEntity.contains(NGSILD_LOCATION_PROPERTY))
+        Assertions.assertEquals(setOf(JSONLD_ID, JSONLD_TYPE), serializedEntity.keys)
+    }
+
+    @Test
+    fun `it should serialize entity with createdAt and modifiedAt if specified`() {
+        val serializedEntity = entity.serializeCoreProperties(true)
+        Assertions.assertTrue(serializedEntity.contains(NGSILD_CREATED_AT_PROPERTY))
+        Assertions.assertTrue(serializedEntity.contains(NGSILD_MODIFIED_AT_PROPERTY))
+        Assertions.assertFalse(serializedEntity.contains("location"))
+    }
+
+    @Test
+    fun `it should serialize entity with location if specified`() {
+        val entityWithLocation = entity.copy(location = "POINT (24.30623 60.07966))")
```

Comment:
review all your point and remove the extra parenthesis

Reply:
the good form is (24.30623 60.07966)?